src/encoded/reports/inequalities.py

```diff
@@ -0,0 +1,53 @@
+import operator
+
+from encoded.reports.serializers import map_string_to_boolean_and_int
+from snovault.elasticsearch.searches.interfaces import COLON
```

Comment:
just for discussion... we are following the pattern that is already in place in Snovault by turning tokens into constants. But many of those are very unlikely to change. Should we continue to use follow it?

Reply:
Yeah, I mainly do this to avoid misspelling strings, probably a little much in this case. The `parse_inequality_param_value` logic actually sort of exists in the snovault PR already (https://github.com/ENCODE-DCC/snovault/blob/240e121f4ec6a2a37700c4d98d3f6dc63cb00fcb/src/snovault/elasticsearch/searches/queries.py#L564) but it gets dumped directly into a dict which wouldn't work here and it didn't seem worth the effort to split it out.